html/benchmarks.js

```diff
@@ -73,31 +79,18 @@ $(function() {
             });
         }
 
-        const tooltips = [];
         const tooltip_formatter = function() {
-            const n_threads = this.x;
-            const tooltip = tooltips[n_threads];
-            if(tooltip)
-                return tooltip;
-
-            const data = [];
-            for(const p of this.points) {
-                const [n_threads, min, max, mean, stdev] = p.series.options.atomic_queue_stats[p.point.index].map(prec0);
-                data[p.series.options.index] = {
-                    name: p.series.name,
-                    color: p.series.color,
-                    min: min,
-                    max: max,
-                    mean: mean,
-                    stdev: stdev,
-                };
-            }
-            const tbody = data.map((d) => `<tr><td style="color: ${d.color}">${d.name}: </td><td><strong>${d.mean}</strong></td><td><strong>${d.stdev}</strong></td><td>${d.min}</td><td>${d.max}</td></tr>`).join('\n')
-            return tooltips[n_threads] = `
-                <span class="tooltip_scalability_title">${n_threads} producers, ${n_threads} consumers</span>
+            const { x, points } = this;
+            const rows = points
+                .filter(({ series }) => series.visible)
```

Comment:
I don't think disabled series elements are present in the points array. Am I wrong?